usl_pipeline/usl_lib/tests/transformers/test_feature_raster_transformers.py

```diff
@@ -0,0 +1,186 @@
+import numpy
+from numpy import testing
+from shapely import geometry
+
+from usl_lib.shared import geo_data
+from usl_lib.transformers import feature_raster_transformers
+
+
+def bbox_polygon(x1: float, y1: float, x2: float, y2: float) -> geometry.Polygon:
+    return geometry.Polygon([(x1, y1), (x2, y1), (x2, y2), (x1, y2), (x1, y1)])
+
+
+def test_transform_to_feature_raster_layers():
+    infiltration_configuration = geo_data.DEFAULT_INFILTRATION_CONFIGURATION
+    max_hydraulic_conductivity = max(
+        [item.hydraulic_conductivity for item in infiltration_configuration.items]
```

Comment:
Can drop the brackets to avoid materializing a list

```python
    max_hydraulic_conductivity = max(
        item.hydraulic_conductivity for item in infiltration_configuration.items
    )
```

might also consider not calculating these are hard-coding the `max_hydraulic_conductivity` value in this test to avoid cascading errors from mis-calculating the max in the tests.

Reply:
Switched to constants.